msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -56,8 +56,31 @@ The app manifest describes how the app integrates into the Microsoft Teams platf
         "color32x32": "%FILENAME-32x32px%"
     },
     "accentColor": "%HEX-COLOR%",
+    "elementRelationshipSet": {
+      "oneWayDependencies" : [
+        {
+          "element" : {
+            "name" : "composeExtensions",
+            "id" : "composeExtension-id",
+            "commandIds": ["command-1-id", "command-2-id"]
```

Comment:
Nit: per previous user testing w/ C4S, user has pointed out a minor inconsistency between line 65's command IDs and line 180 and 198's actual ones below. Supposedly line 65's should be [exampleCmd1, exampleCmd2] to match the ones down below. @erikadoyle